pkg/clustermanager/retrier_client.go

```diff
@@ -12,15 +12,15 @@ import (
 
 // RetrierClient wraps around a ClusterClient, offering retry functionality for some operations.
 type RetrierClient struct {
-	*client
+	*clusterManageClient
```

Comment:
This is just renaming to avoid collision with the client package

Reply:
```suggestion
	*clusterManagerClient
```